docs/getting-started-provider-dev.md

```diff
@@ -0,0 +1,199 @@
+# Getting Started with Cloud Provider OpenStack Development
+
+This guide will help you get started with building a development environment for you 
```

Comment:
How about `This guide will help you build and run a single node Kubernetes cluster with the OpenStack Cloud Provider enabled`?

Reply:
It's a bit of both. It's not just about running the code, but about having it available to develop and test against. We don't have any functional tests enabled, and this is part of the path towards getting those set up.